docs/pages/versions/unversioned/sdk/notifications.md

```diff
@@ -759,6 +760,40 @@ Cancels all scheduled notifications.
 
 A `Promise` resolving once all the scheduled notifications are successfully cancelled or if there are no scheduled notifications.
 
+### `getNextTriggerDateAsync(trigger: NotificationTriggerInput): Promise<number | null>`
+
+Allows you to check what will be the next trigger date for given notification trigger input.
+
+#### Arguments
+
+The notification trigger you would like to check next trigger date for.
+
+> **Note:** Only calendar-based notification triggers are supported, trying to get trigger date for triggers of other types will result in `Promise` rejection with code `ERR_NOTIFICATIONS_INVALID_CALENDAR_TRIGGER`.
```

Comment:
I was thinking about it, but specifying it as an concrete enumerating list we'd have to remember to update the list if we add more notification triggers (eg. `UNLocationNotificationTrigger`-based one) that's why I used a human-understandable non-enumerating list. Trigger types that are not supported are `null` and `ChannelAwareTriggerInput`.

What do you think having read that?

Reply:
From this note, I'm not sure which of: 
```
  | ChannelAwareTriggerInput
  | DateTriggerInput
  | TimeIntervalTriggerInput
  | DailyTriggerInput
  | CalendarTriggerInput
```
are calendar-based without looking at the underlying obj-c and java code. (for example, for all i know a ChannelAwareTriggerInput is calendar based but is just always set to be triggered instantly). 

What if we updated the TS parameter for this method to only accept `DateTriggerInput | TimeIntervalTriggerInput | DailyTriggerInput | CalendarTriggerInput` instead of the more general `NotificationTriggerInput`? the TS type should be harder to forget to update (since the you'd easily spot it during testing)